common-rendering/src/components/Pagination.tsx

```diff
@@ -199,3 +199,5 @@ export const Pagination = ({
 		</Container>
 	);
 };
+
+export default Pagination;
```

Comment:
why is the export changing? our own configs do not allow default exports elsewhere https://github.com/guardian/configs/blob/d42c048b6ab85d1eab6b8f610ff4162aeb63a510/packages/eslint-config/index.js#L46-L47